corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -863,6 +863,23 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
                                 }
 
                                 marker.on('click', () => {
+                                    if (!marker.getPopup()) {
+                                        const caseId = model.get('id');
+                                        const urlObject = formplayerUtils.currentUrlToObject();
+                                        urlObject.addSelection(caseId);
+                                        const fetchingDetails = FormplayerFrontend.getChannel().request("entity:get:details", urlObject, false, false, false, true);
```

Comment:
Doesn't this function only take 4 parameters?
https://github.com/dimagi/commcare-hq/blob/79f02b5e0ec6be0839a230f547cf6be7334162c3/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/api.js#L227

Reply:
Good catch. I just copied another call I found and modified it. @Robert-Costello was there a reason you used 5 arguments [here](https://github.com/dimagi/commcare-hq/blob/23943f827d095de9a25a02703e3ca668fad8348a/corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js#L333)?

Reply:
It was for a flag that indicated that the request was coming from a clickable icon. It should have been removed when we went a different route with case list filtering. If you want you could remove the extra argument here or I can make a separate PR.

Reply:
I removed them